docs/sdk/server-side-sdks/go/go-gettingstarted.md

```diff
@@ -74,3 +68,57 @@ log.Println("DevCycle client not guaranteed to be initialized yet")
 log.Println("Devcycle client initialized")
 ```
 
+## Initialization Options
+
+The SDK exposes various initialization options which can be set when initializing the DevCycle client:
+
+```go
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	devcycle "github.com/devcyclehq/go-server-sdk/v2"
+)
+
+func main() {
+	sdkKey := os.Getenv("DEVCYCLE_SERVER_SDK_KEY")
+
+	options := devcycle.Options{
+		EnableEdgeDB:                 false,
+		EnableCloudBucketing:         false,
```

Comment:
These are defaults - remove?

Reply:
these were there before I touched any of this stuff

Reply:
Yeah, I'm asking if you want to clean up these docs or not

Reply:
the scope of what I'm trying to do here has ballooned a lot. I'd like to stick to where this has gotten to and try to finally get it out and then come back later if there are things we want to clean up further

Reply:
kk